### PR TITLE
add Main, Person. List, Set and Map handlers

### PR DESCRIPTION
**Pull request by brav3luk3**

### CODE REVIEW

File: src/main/ListHandler.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        this.elements = elements;
+        if (isArrayList) {
+            list = new ArrayList<>();
+        } else {
+            list = new LinkedList<>();
+        }
+        this.isArrayList = isArrayList;
+    }
+
+    private static int rnd(int max) {
```
Comment (captim) [inline src/main/ListHandler.java]: normal method names pls

File: src/main/ListHandler.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        return (int) (Math.random() * ++max);
+    }
+
+    private static char randomChar() {
+        int rand = new Random().nextInt(52);
+        char start = (rand < 26) ? 'A' : 'a';
+        return (char) (start + rand % 26);
+    }
+
+    private String listStatus() {
```
Comment (captim) [inline src/main/ListHandler.java]: rename to listType()

File: src/main/Main.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        System.out.println("\n////////////ArrayLIst////////////\n");
+
+        ListHandler arr = new ListHandler(elements, true);
+        arr.add();
+        arr.remove(500);
+        arr.get(8000);
+
+        System.out.println("\n///////////LinkedList////////////\n");
+
+        ListHandler lin = new ListHandler(elements, false);
```
Comment (captim) [inline src/main/Main.java]: you can use one variable "listHandler" and reuse it for second list
ListHandler listHandler = new ListHandler(elements, true);
...
listHandler = new ListHandler(elements, false);
...

File: src/main/Main.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        System.out.println("\n//////////////HashSet////////////\n");
+
+        SetHandler hash = new SetHandler(elements, true);
+        hash.add();
+        hash.remove(500);
+        hash.get(8000);
+
+        System.out.println("\n/////////////TreeSet/////////////\n");
+
+        SetHandler treeSet = new SetHandler(elements, false);
```
Comment (captim) [inline src/main/Main.java]: same as above

File: src/main/Main.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        System.out.println("\n/////////////HashMap//////////////\n");
+
+        MapHandler map = new MapHandler(elements, true);
+        map.add();
+        map.remove(8000);
+        map.get(8000);
+
+        System.out.println("\n/////////////TreeMap//////////////\n");
+
+        MapHandler treeMap = new MapHandler(elements, false);
```
Comment (captim) [inline src/main/Main.java]: same as above

File: src/main/Main.java [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    public static void main(String[] args) {
+
+        int elements = 1_000_000;
+
+        System.out.println("\n////////////ArrayLIst////////////\n");
+
+        ListHandler arr = new ListHandler(elements, true);
+        arr.add();
+        arr.remove(500);
+        arr.get(8000);
```
Comment (captim) [inline src/main/Main.java]: create variable for 500 and 8000 to use it in these methods

File: src/main/MapHandler.java [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+import java.util.TreeMap;
+import java.util.List;
+import java.util.ArrayList;
+
+public class MapHandler {
+    private Map<Integer, Person> map;
+    private Person person;
```
Comment (captim) [inline src/main/MapHandler.java]: Useless field

File: src/main/MapHandler.java [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+    private static int rnd(int max) {
+        return (int) (Math.random() * ++max);
+    }
+
+    private static char randomChar() {
+        int rand = new Random().nextInt(52);
+        char start = (rand < 26) ? 'A' : 'a';
+        return (char) (start + rand % 26);
+    }
```
Comment (captim) [inline src/main/MapHandler.java]: repeated methods in handlers. maybe it's better to create abstract class

Comment (captim) [reply src/main/MapHandler.java]: ... with this methods

File: src/main/MapHandler.java [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+            }
+            person = new Person(personName.toString(), rnd(80));
+            map.put(person.getId(), person);
+        }
+        timeEnd = System.currentTimeMillis();
+
+        System.out.println("time to add " + countElements + " elements to " + mapStatus() + " = " + (timeEnd - timeStart));
+    }
+
+    public void remove(int DelElem) {
```
Comment (captim) [inline src/main/MapHandler.java]: normal names for variables (deleteElements)
*Java code convention

File: src/main/MapHandler.java [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+    public List<Person> getRandomElements(int amount) {
+        List<Person> list = new ArrayList<>();
+        Object[] objects = map.values().toArray();
+        for (int i = 0; i < amount; i++) {
+            list.add((Person) objects[rnd(countElements)]);
+        }
+        return list;
+    }
+
+    public void get(int getElem) {
```
Comment (captim) [inline src/main/MapHandler.java]: same as above

File: src/main/SetHandler.java [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+            }
+            person = new Person(personName.toString(), rnd(80));
+            set.add(person);
+        }
+        timeEnd = System.currentTimeMillis();
+
+        System.out.println("time to add " + countElements + " elements to " + setStatus() + " = " + (timeEnd - timeStart));
+    }
+
+    public void remove(int DelElem) {
```
Comment (captim) [inline src/main/SetHandler.java]: same as above

File: src/main/SetHandler.java [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+    public List<Person> getRandomElements(int amount) {
+        List<Person> list = new ArrayList<>();
+        Object[] objects = set.toArray();
+        for (int i = 0; i < amount; i++) {
+            list.add((Person) objects[rnd(countElements)]);
+        }
+        return list;
+    }
+
+    public void get(int getElem) {
```
Comment (captim) [inline src/main/SetHandler.java]: same as above

**Review by captim — CHANGES_REQUESTED** [review]

**Comment by brav3luk3** [comment]
An abstract class has been created. The MapHandler and ListHandler classes are extended from it.
Methods randomNumber, randomChar and getRandomElement have been changed to a separate class.

**Review by captim — APPROVED** [review]

**Pull request merged**